readme.md

```diff
@@ -61,6 +61,14 @@
 - [Dojo Toolkit](https://github.com/peterkokot/awesome-dojo)
 - [Inspiration](https://github.com/NoahBuscher/Inspire)
 
+## Web development
+
+- [Django](https://github.com/rosarior/awesome-django)
+- [Play1 Framework](https://github.com/PerfectCarl/awesome-play1)
+- [CakePHP](https://github.com/dereuromark/awesome-cakephp)
+- [Symfony2](https://github.com/EmanueleMinotto/awesome-symfony2)
+- [Laravel](https://github.com/tuwannu/awesome-laravel)
+- [Learn to Program](https://github.com/karlhorky/learn-to-program)
```

Comment:
@sindresorhus The resources are mostly based on web development and web technologies. The title is maybe not the best. Maybe if we put an annotation in parentheses?

- [Learn to Program](https://github.com/karlhorky/learn-to-program) (Web Dev foundation)

Or would you add it to another section?